appveyor.yml

```diff
@@ -37,7 +37,7 @@ install:
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
 
   # Install the build and runtime dependencies of the project.
-  - "%CMD_IN_ENV% pip install -v %WHEELHOUSE% -r tools/appveyor/requirements.txt"
+  - "%CMD_IN_ENV% pip install --trusted-host 28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com -r tools/appveyor/requirements.txt"
```

Comment:
I don't think so, wheels are the way forward. 



—
Sent from Mailbox

On Sun, May 24, 2015 at 6:14 AM, Arve Seljebu <notifications@github.com>
wrote:

>> @@ -37,7 +37,7 @@ install:
>>    - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
>>  
>>    # Install the build and runtime dependencies of the project.
>> -  - "%CMD_IN_ENV% pip install -v %WHEELHOUSE% -r tools/appveyor/requirements.txt"
>> +  - "%CMD_IN_ENV% pip install --trusted-host 28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com -r tools/appveyor/requirements.txt"
> Any reason why both wheel and `bdist_wininst` is built/packaged?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/scikit-image/scikit-image/pull/1526/files#r30951835